src/sentry/notifications/serializers.py

```diff
@@ -32,3 +33,17 @@ def serialize(
         **kwargs: Any,
     ) -> Mapping[str, Optional[str]]:
         return super().serialize(obj, **kwargs)
+
+
+class NotificationSettingsProviderSerializer(NotificationSettingsBaseSerializer):
+    def serialize(
+        self,
+        obj: NotificationSettingProvider,
+        *args: Any,
+        **kwargs: Any,
+    ) -> Mapping[str, Optional[str]]:
+        output = super().serialize(obj, **kwargs)
+        return {
+            **output,
```

Comment:
you don't have to unpack output, you can modify output and return it, may be more legible.

Reply:
@JoshFerge mypy was screaming at me when I did that

Reply:
i see, i suppose you'd have to cast it too. how you have makes sense.